app/boardgame-tournaments/_components/FeaturedBoardgameTournaments/index.tsx

```diff
@@ -207,15 +200,13 @@ const FeaturedBoardgameTournaments = ({
                     playerCount ? `${playerCount} Participants` : "Loading"
                   }
                 />
-
                 <ul className="pt-2 font-Rubik-Light">
-                  {players?.map((player: any, index) => {
-                    return (
-                      <li key={`player-${index}`} className="name">
-                        {player?.name}
-                      </li>
-                    );
-                  })}
+                  {players?.map((player: any, index: any) => (
+                    <li key={index} className="name">
+                      {player?.name}
+                    </li>
+                  ))}
+                  {players?.length >= 13 && <li className="name">...</li>}
```

Comment:
good convention is 

```suggestion
                  {players?.length >= 13 ? <li className="name">...</li> : null}
```

otherwise if its false, its more readable what to expect